### PR TITLE
fix(env-checker): fix ENOENT error when run dotnet-install script with execFile

### PR DESCRIPTION
**Pull request by a1exwang**
- When using relative PATH in execFile, it will throw ENOENT on macOS 11.2.3. Changed to absolute path to workaround this issue
## Tests:
Run local debug without .NET SDK

#### Output Channel
image
#### Final result
image

#### Output Channel
image
#### Final result
image

### Linux
Linux is not affected because we won't install .NET SDK on Linux

### CODE REVIEW

**Comment by kimizhu** [comment]
@a1exwang can you test on Windows/Mac/Linux before checking the code? You can add the test result here to let us know.

File: packages/fx-core/src/plugins/resource/function/utils/depsChecker/dotnetChecker.ts [hunk 1]
```diff
@@ -394,7 +394,7 @@ export class DotnetChecker implements IDepsChecker {
   ): Promise<string[]> {
     const command = [
       // path.join does not prepend '.'
```
Comment (qinezh) [inline packages/fx-core/src/plugins/resource/function/utils/depsChecker/dotnetChecker.ts:396]: this comment can be removed.

**Review by qinezh — APPROVED** [review]

**Comment by a1exwang** [comment]
> @a1exwang can you test on Windows/Mac/Linux before checking the code? You can add the test result here to let us know.

Tested on Windows and macOS. Linux is not affected because we won't install .NET SDK on Linux

File: packages/fx-core/src/plugins/resource/function/utils/depsChecker/dotnetChecker.ts (hunk 1 above)
Comment (a1exwang) [reply packages/fx-core/src/plugins/resource/function/utils/depsChecker/dotnetChecker.ts]: Removed

**Pull request merged**